media/css/sandstone/buttons.less

```diff
@@ -333,6 +333,15 @@ ul.download-list {
     display: block !important;
 }
 
+// Add another rule for old browsers where :not isn't supported. If this is
+// mixed with the rule above, the entire rule will be invalid on those legacy
+// environments and some tests will fail.
+.osx:not(.pre-mavericks) .download-button .os_osx,
+.osx.pre-mavericks .download-button:not(.mavericks-required) .os_osx {
+    display: block !important;
+}
```

Comment:
[Can i use says Fx 3 doesn't support CSS3 selectors](http://caniuse.com/#feat=css-sel3). I would assume MDN is more accurate though?